applications/fax/src/fax_worker.erl

```diff
@@ -879,11 +876,26 @@ prepare_contents(<<"image/", SubType/binary>>, JobId, RespContent, TmpDir) ->
     end;
 
 prepare_contents(<<?OPENXML_MIME_PREFIX, _/binary>> = CT, JobId, RespContent, TmpDir) ->
-    convert_openoffice_document(CT, TmpDir, JobId, RespContent);
+    Secret = ?CONVERTAPI_SECRET,
+    case Secret of
+        'undefined' -> convert_openoffice_document(CT, TmpDir, JobId, RespContent);
+        _ -> convert_msoffice_document(CT, TmpDir, JobId, RespContent, Secret)
+    end;
+
+prepare_contents(<<?OPENOFFICE_MIME_PREFIX, _/binary>> = CT, JobId, RespContent, TmpDir) ->
```

Comment:
Added openoffice format conversion.
May other formats is supported by [ConvertAPI service](https://www.convertapi.com/) but many is died and unknown.
Additional mime format is added as need.